base/cmd.jl

```diff
@@ -14,8 +14,8 @@ struct Cmd <: AbstractCmd
     env::Union{Vector{String},Nothing}
     dir::String
     cpus::Union{Nothing,Vector{UInt16}}
-    Cmd(exec::Vector{String}) =
-        new(exec, false, 0x00, nothing, "", nothing)
+    Cmd(exec::Vector{<:AbstractString}) =
+        new(string.(exec), false, 0x00, nothing, "", nothing)
```

Comment:
This call is defined to be a no-op. Did you mean `map(String, exec)`? Or just let the existing implied `convert(Vector{String}, exec)` take care of it?
```suggestion
        new(exec, false, 0x00, nothing, "", nothing)
```

Reply:
Oh duh, yeah I suppose I don't need it here.